VideoLocker/src/main/java/org/edx/mobile/services/EdxCookieManager.java

```diff
@@ -53,8 +53,6 @@ public void onReceiveValue(Object value) {
         }
         PrefManager pref = new PrefManager(MainApplication.instance(), PrefManager.Pref.LOGIN);
         pref.put(PrefManager.Key.AUTH_ASSESSMENT_SESSION_ID, "");
-        //we can not get the expiration date from cookie, so just set it to expire for one day
-        pref.put(PrefManager.Key.AUTH_ASSESSMENT_SESSION_EXPIRATION, -1);
```

Comment:
Actually, we might want to retain this logic if we want to set an arbitrary interval (one hour) as the expiration period?